merlin/dag/base_operator.py

```diff
@@ -409,3 +410,92 @@ def _get_columns(self, df, selector):
             return df[selector.names]
+
+    @property
+    def export_name(self):
+        """
+        Provides a clear common english identifier for this operator.
+
+        Returns
+        -------
+        String
+            Name of the current class as spelled in module.
+        """
+        return self.__class__.__name__.lower()
+
+    def export(self, path: str, input_schema: Schema, output_schema: Schema, **kwargs):
+        """
+        Export the class object as a config and all related files to the user defined path.
+
+        Parameters
+        ----------
+        path : str
+            Artifact export path
+        input_schema : Schema
+            A schema with information about the inputs to this operator.
+        output_schema : Schema
+            A schema with information about the outputs of this operator.
+        params : dict, optional
+            Parameters dictionary of key, value pairs stored in exported config, by default None.
+        node_id : int, optional
+            The placement of the node in the graph (starts at 1), by default None.
+        version : int, optional
+            The version of the model, by default 1.
+
+        Returns
+        -------
+        model_config: dict
+            The config for the exported operator (Triton model).
+        """
+        raise NotImplementedError(
+            "Exporting an operator to run in a particular context (i.e. Triton) "
+            "only makes sense when a runtime is specified. To select an "
+            f"operator for the appropriate runtime, replace {self.__class__.__name__} "
+            f"with a runtime-specific operator class, possibly {self.__class__.__name__}Triton"
+        )
+
+    @classmethod
+    def from_model_registry(
+        cls, registry: merlin.dag.utils.ModelRegistry, **kwargs
+    ) -> "BaseOperator":
+        """
+        Loads the Operator from the provided ModelRegistry.
+
+        Parameters
+        ----------
+        registry : ModelRegistry
+            A ModleRegistry object that will provide the path to the model.
+        **kwargs
+            Other kwargs to pass to your Operator's constructor.
+
+        Returns
+        -------
+        Operator
+            New node for graph.
+        """
+
+        return cls.from_path(registry.get_artifact_uri(), **kwargs)
+
+    @classmethod
+    def from_path(cls, path, **kwargs) -> "BaseOperator":
+        """
+        Loads the Operator from the path where it was exported after training.
+
+        Parameters
+        ----------
+        path : str
+            Path to the exported model.
+        **kwargs
+            Other kwargs to pass to your Operator's constructor.
+
+        Returns
+        -------
+        InferenceOperator
+            New node for graph.
+        """
+        raise NotImplementedError(f"{cls.__name__} operators cannot be instantiated with a path.")
+
+    # TODO: This operator should be moved once all triton specific op migrations completed
```

Comment:
Definitely drop this part, which I'm hoping we don't even need in Systems anymore